pom.xml

```diff
@@ -104,8 +104,8 @@
         <elytron.version>1.15.3.Final</elytron.version>
         <elytron.undertow-server.version>1.9.0.Final</elytron.undertow-server.version>
         <jetty92.version>9.2.4.v20141103</jetty92.version>
-        <jetty93.version>9.3.9.v20160517</jetty93.version>
-        <jetty94.version>9.4.29.v20200521</jetty94.version>
+        <jetty93.version>9.3.29.v20201019</jetty93.version>
+        <jetty94.version>9.4.35.v20201120</jetty94.version>
```

Comment:
But for Jetty 9.4 the latest available seems to be [_9.4.40.v20210413_](https://www.eclipse.org/jetty/download.php). Is there a reason why that one wasn't used?

Since per the [Jetty security page](https://www.eclipse.org/jetty/security_reports.php) there are more CVEs fixed in _9.4_ till _9.4.39_.

If _9.4.40.v20210413_ isn't suitable, can we use _9.4.39_ instead?

Reply:
That's a great point @iankko, updated my PR.